src/middleware/auth/base-auth.ts

```diff
@@ -43,7 +30,7 @@ export abstract class AbstractAuthHandler implements IAuthResourceHandler
         this.namespace = '' as Namespaces
         this.token = '' as string
         this.scopes = []
-        this.customer_id = '' as string
+        this.customerId = '' as string
```

Comment:
Good catch! This is a great example of making naming conventions consistent (and following TypeScript style guidelines)